metaflow/util.py

```diff
@@ -137,6 +138,9 @@ def get_username():
     could not be determined.
     """
     # note: the order of the list matters
+    if METAFLOW_USER:
```

Comment:
Is there a difference between the `METAFLOW_USER` defined here and the one is `ENVVARS`? If not, then remove the duplicate.

Reply:
METAFLOW_USER may come from metaflow config file instead of env vars in this case.

Reply:
true it comes from `metaflow_config.py`